core/trino-main/src/test/java/io/trino/sql/query/TestOrderedAggregation.java

```diff
@@ -167,4 +167,31 @@ public void testRepeatedSortItems()
         assertThat(assertions.query("SELECT count(x ORDER BY y, y) FROM (VALUES ('a', 2)) t(x, y)"))
                 .matches("VALUES BIGINT '1'");
     }
+
+    @Test
+    public void testCompletelyFilteredGroup()
+    {
+        // This query filters out all values to a most groups, which results in an accumulator with no pages to sort.
+        // This can cause a failure if the ordering code does not inform the accumulator of the max row group.
+        assertThat(assertions.query("" +
+                "select count(id) > 15000, sum(cardinality(v))" +
+                "from (" +
```

Comment:
add a space

```suggestion
                "select count(id) > 15000, sum(cardinality(v)) " +
                "from (" +
```

so that `from` is separated from `sum(cardinality(v))`


also, can you please uppercase the sql keywords?